ecc-hol/workshops/sandbox/manifest.json

```diff
@@ -1,48 +1,68 @@
 {
-    "workshoptitle": "The Title of the Workshop is defined in the manifest",
-    "help": "livelabs-help-**enterarea**_us@oracle.com",
+    "workshoptitle": "Oracle E-Business Suite Enterprise Command Center - From Installation to Personalization",
+    "help": "livelabs-help-oci_us@oracle.com",
     "variables": ["../../variables/variables.json",
-        "../../variables/variables-in-another-file.json"],
+      "../../variables/variables-in-another-file.json"],
     "tutorials": [
-        {
-            "title": "Get Started",
-            "description": "Prerequisites for LiveLabs (Oracle-owned tenancies). The title of the lab and the Contents Menu title (the title above) match for Prerequisite lab. This lab is always first.",
-            "filename": "https://oracle-livelabs.github.io/common/labs/cloud-login/cloud-login-livelabs2.md"
-        },
-        {
-            "title": "Introduction",
-            "description": "The Introduction is always second for LiveLabs. The title and contents menu title match for the Introduction.",
-            "filename": "../../introduction/introduction.md"
-        },
-        {
-            "title": "Lab 1: Provision an Instance",
-            "description": "Labs that follow the introduction are numbered, starting with Lab 1",
-            "filename": "../../provision/provision.md"
-        },
-        {
-            "title": "Lab 2: Setup the Environment",
-            "filename": "../../setup/setup.md"
-        },
-        {
-            "title": "Lab 3: Load Data into an Instance",
-            "filename": "../../data-load/data-load.md"
-        },
-        {
-            "title": "Lab 4: Query Data",
-            "filename": "../../query/query.md"
-        },
-        {
-            "title": "Lab 6: Variables in LiveLabs",
-            "filename": "../../variables/variables.md"
-        },
-        {
-            "title": "Lab 5: Tables in LiveLabs",
-            "filename": "../../tables/tables.md"
-        },
-        {
-            "title": "Need Help?",
-            "description": "Solutions to Common Problems and Directions for Receiving Live Help",
-            "filename":"https://oracle-livelabs.github.io/common/labs/need-help/need-help-livelabs.md"
-        }
+      {
+        "title": "Get started",
+        "description": "The Introduction is always second for LiveLabs. The title and contents menu title match for the Introduction.",
+        "filename": "../../query/Getstarted.md"
```

Comment:
@ecchol - Please use the common labs link -
Desktop version - https://oracle-livelabs.github.io/common//labs/remote-desktop/using-novnc-remote-desktop.md
 Sandbox version - https://oracle-livelabs.github.io/common//labs/cloud-login/cloud-login-livelabs2.md
Tenancy version - https://oracle-livelabs.github.io/common//labs/cloud-login/cloud-login.md

Reply:
Also, delete the Get Started and Need Help lab in query folder.

Reply:
updated the file as per comments. Thank you